README.md

```diff
@@ -14,3 +14,18 @@ make install
 Then you can run the scripts in the script folder in sequence.
 
 
+For android
+
+```java
+public class Application extends android.app.Application {
+	@Override public void onCreate() {
+		MKApplication.onCreate(this);
+	}
+}
+```
+
+# Compile for Android
+To compile for Android use `./gradlew assembleAndroidRelease` and the output `aar` will be in `./lib/build/outputs/aar/lib-android-release.aar`
+
+# Compile for Java
```

Comment:
@xanscale asked whether we should expose this command in the README.md. I would mention that this is a possibility but that we DO NOT directly support this use case.